mappings/net/minecraft/world/gen/feature/size/FeatureSize.mapping

```diff
@@ -1,8 +1,13 @@
 CLASS net/minecraft/class_5201 net/minecraft/world/gen/feature/size/FeatureSize
+	COMMENT In charge of determining the amount of space needed by a tree to generate.
```

Comment:
Is it only used by tree features?

Reply:
Yeah, it's only used by trees at this time.